lib/situation_room/site_controller.ex

```diff
@@ -0,0 +1,89 @@
+defmodule SituationRoom.Site.Controller do
+  @moduledoc """
+  Documentation for `SituationRoom.Site.Controller`.
+  """
+  alias SituationRoom.Site
+  use Plug.Router
+
+  plug(:match)
+
+  plug(Plug.Parsers,
+    parsers: [:urlencoded, :multipart],
+    pass: ["text/*"]
+  )
+
+  plug(:dispatch)
+
+  # Get all sites
+  get "/" do
+    send_resp(
+      conn,
+      200,
+      "[#{for content <- Site.get_all_sites(), do: "#{build_site_resp(content)}, "}]"
```

Comment:
This kind of thing should not be "inlined" in a `send_resp/3` call.  I'd expect us to create a var to hold the value of what `Sites.get_all_sites() returns.  And then can't we just do something like `Jason.encode!(sites)` and return the json directly?

Reply:
I was going to ask you about this - I was unsure if adding another dependency for only this use case was okay. I know my implementation right here is ugly to say the least.